src/sentry/api/event_search.py

```diff
@@ -893,21 +910,23 @@ def resolve_field(field):
     if not isinstance(field, six.string_types):
         raise InvalidSearchQuery("Field names must be strings")
 
-    if field in FIELD_ALIASES:
-        special_field = deepcopy(FIELD_ALIASES[field])
-        return (special_field.get("fields", []), special_field.get("aggregations", []))
+    sans_parens = field.strip("()")
+    if sans_parens in FIELD_ALIASES:
```

Comment:
Would having both `p95` and `p95()` in the FIELD_ALIASES map make this logic simpler?

Reply:
No I don't think so. This check is largely here to track whether or not we removed brackets before sending to snuba, so we can add them back when returning the results to the frontend.

Reply:
Turns out I didn't even need the check, the frontend strips the brackets anyways.